### PR TITLE
Adds Ollama chat model and llm

### PR DESCRIPTION
**Pull request by jacoblee93**

### CODE REVIEW

File: langchain/src/util/ollama.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+export async function* createOllamaStream(
+  baseUrl: string,
+  params: OllamaRequestParams,
+  options: OllamaCallOptions
+): AsyncGenerator<OllamaGenerationChunk> {
+  const response = await fetch(`${baseUrl}/api/generate`, {
+    method: "POST",
+    body: JSON.stringify(params),
+    headers: {
```
Comment (jmorganca) [inline langchain/src/util/ollama.ts:59]: small note: you shouldn't need this header since it returns a stream of events (vs json)

**Comment by jmorganca** [comment]
Looks great!

File: langchain/src/util/ollama.ts (hunk 1 above)
Comment (jacoblee93) [reply langchain/src/util/ollama.ts:59]: Wouldn't it refer to the request body's content type rather than the response?

Comment (jmorganca) [reply langchain/src/util/ollama.ts:59]: That's right, sorry! Misread this as the `Accept` header.

**Pull request merged**